Assets/Scenes/Hilly plane level.unity.orig

```diff
@@ -0,0 +1,3861 @@
+%YAML 1.1
```

Comment:
I believe this file was created by the merge tool and we don't need it. Lets remove it and the corresponding `.meta` file

Reply:
done